docs/workflow-migration.md

```diff
@@ -43,7 +43,8 @@ Note: Github Actions doesn't suppport regex filtering of branch filters, so it u
 ### Deploy
 This will create a deployment activity for each corresponding environment within `config.yml`
 
-Approval gates are configured within the repository settings under **environments**, so aren't included within the deployment here,
+Approval gates (a.k.a. **manual approval**, or **approvals** in CircleCI) are configured within the repository itself, so aren't included within the deployment here.  They are set per repository, and can be access via the Web UI at **environments > (env name) > Required reviewers** and ensuring this checkbox is checked.  See GitHub's ["Deployments and environments"](https://docs.github.com/en/actions/reference/workflows-and-actions/deployments-and-environments#required-reviewers) documentation for further information.
+
```

Comment:
would be worth mentioning that the github UI configuration shouldn't be changed directly, instead people should amend their cloud platform github configuration to set the values e.g. https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-prisoner-location-prod/resources/hmpps-prisoner-location-github.tf#L15